models/tools/setup_dev/proxy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+echo '# Proxy setting' >> /etc/profile
+echo 'export http_proxy="http://10.112.1.184:8080/"' >> /etc/profile
+echo 'export https_proxy="http://10.112.1.184:8080/"' >> /etc/profile
+echo 'export no_proxy="localhost,127.0.0.1,165.213.149.200,10.113.136.32,github.sec.samsung.net"' >> /etc/profile
```

Comment:
Let's remove this

Reply:
Nice catch. I updated the statements instead of removing the statements because users will be developers that are working  in a company. :)